sbin/build.sh

```diff
@@ -199,9 +199,10 @@ getOpenJdkVersion() {
 
 # Ensure that we produce builds with versions strings something like:
 #
-# openjdk version "1.8.0_131"
-# OpenJDK Runtime Environment (build 1.8.0-temurin-<user>_2017_04_17_17_21-b00)
-# OpenJDK 64-Bit Server VM (build 25.71-b00, mixed mode)
+# openjdk 11.0.12 2021-07-20
+# OpenJDK Runtime Environment Temurin-11.0.12+7 (build 11.0.12+7)
+# OpenJDK 64-Bit Server VM Temurin-11.0.12+7 (build 11.0.12+7, mixed mode)configureVersionStringParameter() {
```

Comment:
Is this trailing `configureVersionStringParameter() {` intentional?

Reply:
Ah that explains something I hit earlier - no it's not - will fix - well spotted :-)